sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -348,11 +348,11 @@ case class NotLikeAny(child: Expression, patterns: Seq[UTF8String]) extends Like
     Examples:
       > SET spark.sql.parser.escapedStringLiterals=true;
       spark.sql.parser.escapedStringLiterals	true
-      > SELECT '%SystemDrive%\Users\John' _FUNC_ '%SystemDrive%\\Users.*';
+      > SELECT _FUNC_('%SystemDrive%\Users\John', '%SystemDrive%\\Users.*');
```

Comment:
hm, since RLIKE/LIKE syntaxes are already documented in [the SQL doc](https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-like.html) , how about updating the usage (`str _FUNC_ regexp` -> `_FUNC_(str, regext)`) accordingly?

Reply:
`[ NOT ] { LIKE search_pattern [ ESCAPE esc_char ] | [ RLIKE | REGEXP ] regex_pattern }` is a syntax, but `_FUNC_(str, regext)` is a function call.

Reply:
What do you mean? I meant the current generated doc is incorrect (`str regexp_like regex` in the usage);

<img width="500" alt="Screen Shot 2020-12-08 at 22 14 34" src="https://user-images.githubusercontent.com/692303/101488973-9e1bb300-39a3-11eb-92bc-9d0f654afae4.png">

```
scala> sql("select 'aaa' regexp_like 'aaa'").show()
org.apache.spark.sql.catalyst.parser.ParseException:
extraneous input ''aaa'' expecting {<EOF>, ';'}(line 1, pos 25)

== SQL ==
select 'aaa' regexp_like 'aaa'
-------------------------^^^
```

Reply:
And, the column name below looks weird;
```
scala> sql("select regexp_like('aaa', 'aaa')").show()
+-------------------+                                                           
|aaa REGEXP_LIKE aaa| --> should be REGEXP_LIKE(aaa, aaa)?
+-------------------+
|               true|
+-------------------+
```

Reply:
I understand your mean now.